css/events.css

```diff
@@ -161,6 +161,28 @@ Section Styles (background, padding, etc.)
 	margin: 0 auto;
 }
 
+.event-detail-container {
+	display: flex;
+	flex-flow: column;
+	justify-content: space-around;
+	align-items: center;
+	gap: var(--spacing-xl);
+	padding: var(--spacing-xl);
+}
+
+.event-detail-container img {
+	width: 50%;
```

Comment:
on a lot of viewports, the img is very large and goes beyond the the user's view. I think playing around with constraints can help with this.

desktop view:

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/f3578d78-c3f4-4bb8-b581-0fa0c3a45439)

mobile view

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/cfe5dab0-dbda-405c-8b79-cd082873ce23)